tests/examples/example_SBT_Hi_T.out

```diff
@@ -0,0 +1,224 @@
+                               *****************
+                               ***CASE REPORT***
+                               *****************
+
+Simulation Metadata
+----------------------
+ GEOPHIRES Version: 3.5.2
+ Simulation Date: 2024-09-02
+ Simulation Time:  13:59
+ Calculation Time:     14.837 sec
+
+                           ***SUMMARY OF RESULTS***
+
+      End-Use Option: Electricity
+      Average Net Electricity Production:                     9.62 MW
+      Electricity breakeven price:                           18.23 cents/kWh
+      Number of production wells:                             1
+      Number of injection wells:                              1
+      Flowrate per production well:                          80.0 kg/sec
+      Well depth (or total length, if not vertical):          5.8 kilometer
+      Geothermal gradient:                                     60 degC/km
```

Comment:
Is this accurate/expected? Seems like very low output for that depth, gradient, and flow rate...